airflow-core/src/airflow/ui/src/i18n/locales/zh_TW/dashboard.json

```
@@ -0,0 +1,39 @@
+{
+  "welcome": "歡迎",
+  "stats": {
+    "stats": "統計",
+    "failedDags": "失敗的 DAGs",
+    "runningDags": "執行中的 DAGs",
+    "activeDags": "活躍的 DAGs",
```

```suggestion
    "activeDags": "啟用中的 Dags",
```

I think it means it's now being activated. Translation is hard 🤦‍♂️

---

Does “active” mean _not paused_? I think your translation is appropriate if so.

---

ah... right...